app/src/main/java/org/mozilla/fenix/perf/Performance.kt

```diff
@@ -37,4 +45,25 @@ object Performance {
             activity.reportFullyDrawn()
         }
     }
+
+    /**
+     * Processes intent for Performance testing to remove protection pop up ( but keeps the TP
+     * on) and removes the onboarding screen.
+     */
+    fun processIntentIfPerformanceTest(intent: Intent, context: Context) {
+        if (!intent.getBooleanExtra(EXTRA_IS_PERFORMANCE_TEST, true)) {
```

Comment:
I think this got inverted in the last PR: this should be false, right? Because it's inverted by `!`